src/Form/Control/ScopeBuilder.php

```diff
@@ -146,6 +157,13 @@ class ScopeBuilder extends Control
         self::OPERATOR_NOT_EMPTY,
     ];
 
+    protected const ENUM_OPERATORS = [
+        self::OPERATOR_EQUALS,
+        self::OPERATOR_DOESNOT_EQUAL,
```

Comment:
I think `OPERATOR_DOESNOT_EQUAL` and `OPERATOR_NOT_EMPTY` should be in same "formatting".
I mean both `OPERATOR_NOT_something` like `OPERATOR_NOT_EQUAL` even if that's maybe against spelling rules (not sure)